src/writer/creatordata.h

```diff
@@ -59,7 +59,7 @@ namespace zim
         typedef std::vector<std::string> MimeTypesList;
         typedef std::vector<Cluster*> ClusterList;
         typedef Queue<Cluster*> ClusterQueue;
-        typedef Queue<Task*> TaskQueue;
+        typedef Queue<std::shared_ptr<Task>> TaskQueue;
```

Comment:
Because the smart pointer is store in the generic `Queue`.
And this `Queue` is also used to pass cluster to `clusterWriter` worker.
And it is using `getHead` which copy (not move) the head value of the queue.

We can probably adapt the queue depending if we pass a `unique_ptr` or something else but I think it is simpler to state that we need a copyable object to store it in the queue.